### ISSUE TITLE
ADA Compliance - Title meta tag for menu links 

### ISSUE THREAD

**Issue by ljgarnes** [elementor/elementor#10100]
I have a website where ADA Compliance is very important. Website Accessibility

An empty link was encountered. If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users. Switch to the Text view to locate and remove the empty tag or add an aria-label attribute with descriptive text. If the link includes an image that is missing alternate text, use the edit image option to add alt text to the image. Upgrade to the full version for additional options to automatically correct this issue. WCAG 2.1 (Level A) - 2.4.4

Can you add a feature for ALL links where we can include a Title Attribute or Link Label? Keep scrolling down for the example.
![Screenshot_20200109-034620_Outlook](https://user-images.githubusercontent.com/36168330/72052170-a0a4f280-3292-11ea-883f-001cb45ecbc5.jpg)

**Comment by rami-elementor** [comment]
This feature already exist in Elementor.

Click the cog icon to add custom attributes:
![control-url-options2](https://user-images.githubusercontent.com/92088692/215492349-09d07d2b-a8f7-44b8-8b85-ee649ca8bd2e.png)

You can add comma-separated `key|value` pairs. For example: ` title|Some title , aria-label|Some label `
